staging/src/k8s.io/apiserver/pkg/util/peerproxy/peerproxy.go

```diff
@@ -36,32 +41,26 @@ type Interface interface {
 
 // New creates a new instance to implement unknown version proxy
 func NewPeerProxyHandler(informerFactory kubeinformers.SharedInformerFactory,
-	svm storageversion.Manager,
-	proxyTransport http.RoundTripper,
 	serverId string,
 	reconciler reconcilers.PeerEndpointLeaseReconciler,
-	serializer runtime.NegotiatedSerializer) *peerProxyHandler {
+	ser runtime.NegotiatedSerializer,
+	loopbackClientConfig *rest.Config,
+	proxyClientConfig *transport.Config) *peerProxyHandler {
```

Comment:
Since this is a backward incompatible change to an exported function in published (staging) code-  Am I correct in assuming that the're really no backward compatible way of doing this unless we keep the whole old implementation around and a new one next to it?  I think I can live with that for a alpha level feature, but wanted to call it out.

We should probably add something to the Godoc for this function warning that it's alpha functionality and subject to change?

Reply:
+1 on alpha functionality warning on the godocs.

Reply:
Done